### DISCUSSION TITLE
Invitation to repository is going to a 404 page when accepted

### DISCUSSION THREAD

**Discussion by David-H-Bolton** [community/community#66454] [New to GitHub] [answered]
A colleague has set up a private repository and given me an invite. But when I click Accept or View Invitations, I get a 404 page. I've copied and pasted the link and it gives the same. I received the invitation last night, good for seven days it said.

**Comment by queenofcorgis** [comment]
Hey there 👋🏼 

Are you still experiencing this?

**Reply by David-H-Bolton** [reply]
Yes. In fact today I created a repository myself and added my friend as a collaborator and sent an invite. He got a 404 when clicking the link in the invite.

**Comment by David-H-Bolton** [comment]
Yes. I just tried it with the invite link sent Thursday September 7.

**Reply by queenofcorgis** [reply]
Are you able to ask the project owner for an invite link?  They can go to the repository - settings tab - collaborators on the side panel and then copy invite link. 

Make sure you're logged into GitHub once you receive the link then paste it into your browser to accept.

**Reply by David-H-Bolton** [reply, marked as answer]
It has been resolved. It was a stupid thing. I discovered I had setup a different GitHub account maybe ten years ago and never used it and somehow the invitation was going to that one. As soon as the invitation went out to the correct one it worked.  The accounts were set up on different email addresses.

**Reply by queenofcorgis** [reply]
Oh wow - I'm glad you were able to find the cause and fix it. Appreciate you coming back to share that!

**Comment by David-H-Bolton** [comment]
I'll ask him to send another one. Thanks.

**Comment by David-H-Bolton** [comment]
He just tried it and it produces the same result.
https://github.com/watersmeister/TALdev-/invitations

[image: image.png]

**Reply by queenofcorgis** [reply]
Thanks for the updates! Are you part of the same organization? If so, can you confirm your account is properly authenticated?

**Comment by 2004seraph** [comment]
Hi, I have this issue, I cannot even get to the /invitations page, it gives me a 404 despite me being invited.

**Reply by yangdavid195** [reply]
Running into this same issue, I'm clicking the View invitation link from my email, made sure I'm logged into the correct github account, but still getting a 404 error.
 
Anyone else have experience with this, or been able to resolve this issue?

**Reply by ysfbsf** [reply]
I'm facing the same issue

**Reply by yangdavid195** [reply]
Update on this: I had my email address set to private, so I unchecked that setting and had the colleague resend the invite and it worked. Not sure why/if that was the solution but that worked for me.
